lib/D2L.Extensibility.AuthSdk.UnitTests/FunctionalApiUriTests.cs

```diff
@@ -288,6 +288,32 @@ public class FunctionalApiUriTests {
 		}
 
 		[Test]
+		public void UserContext_CreateAuthUri_QuerystringIsMaintained() {
+			Uri queryUrl = m_userContext.CreateAuthenticatedUri( TestConstants.API_PATH_WITH_QUERY, "GET" );
+
+			Assert.IsTrue(
+				queryUrl.ToString().Contains( TestConstants.API_PATH_QUERYSTRING ),
+				"The querystring parameter was not present"
+				);
+		}
+
+		[Test]
+		public void UserContext_CreateAuthUri_MultipleQuerystringsAreNotPresent() {
+			const string querystringCharacterEncoded = "%3F";
+
+			Uri queryUrl = m_userContext.CreateAuthenticatedUri( TestConstants.API_PATH_WITH_QUERY, "GET" );
+			bool duplicatedQuery = (

```

Comment:
This test doesn't exactly seem to do what it's name implies.

Also don't believe this would have failed without the fix in place. Does it? (It would have _only_ had the encoded querystring, thus duplicatedQuery would still be false).

I think the test above of ("properly includes query params which were part of the provided path") catches the case in a meaningful enough way.